%YAML:1.0

# ros parameters
image0_topic: '/mynteye/left/image_raw'
image1_topic: '/mynteye/right/image_raw'

# cameras parameters
undistort: 1

# camera0 intrinsics
camera0.fx: 3.6509624680337657e+02
camera0.fy: 3.6582107844466208e+02
camera0.cx: 3.7731877674972537e+02
camera0.cy: 2.3804064937753677e+02
camera0.k1: -2.9837705601814912e-01
camera0.k2: 7.5700979548022657e-02
camera0.p1: 1.7530605999019629e-04
camera0.p2: -5.4284812981307030e-04

# camera1 intrinsics
camera1.fx: 3.6401748092027668e+02
camera1.fy: 3.6417746726496523e+02
camera1.cx: 3.7559636500280993e+02
camera1.cy: 2.4903879600712332e+02
camera1.k1: -3.0803062161957084e-01
camera1.k2: 8.3487670591387161e-02
camera1.p1: -7.3109575503709556e-05
camera1.p2: -1.3285231720891767e-03

# body_to_cam0 is inverse of [R T]
body_to_cam0: !!opencv-matrix
   rows: 4
   cols: 4
   dt: d
   data: [  1,     0,     0,     0,
            0,     1,     0,     0,
            0,     0,     1,     0,
            0,     0,     0,     1]


# body_to_cam1 is inverse of [R T]
body_to_cam1: !!opencv-matrix
   rows: 4
   cols: 4
   dt: d
   data: [  1,    0,    0,    0.1219,
            0,    1,    0,    0.0004,
            0,    0,    1,   -0.0034,
            0,    0,    0,    1.0000]

